src/main/scala/outwatch/dom/helpers/DomUtils.scala

```diff
@@ -201,12 +199,22 @@ object DomUtils {
     }
   }
 
+
+  private final case class ChildrenNodes(
+    children: List[VNode_] = Nil,
+    hasStreams: Boolean = false
+  )
+  private def extractChildren(nodes: Seq[ChildVNode]): ChildrenNodes = nodes.foldRight(ChildrenNodes()) {
+    case (vn: VNode_, cn) => cn.copy(children = vn :: cn.children)
+    case (_, cn) => cn.copy(hasStreams = true)
```

Comment:
This seems a bit dangerous, as we are only assuming that the type of `_` must be a stream and then set `hasStreams` to true. If someone would refactor the inheritance of `ChildVNode`, this usage not be catched by the compiler.

Reply:
Yes, that's correct. I've updated the PR.